### ISSUE TITLE
Thanks for the code. Have you reproduced the results in the paper?

### ISSUE THREAD

**Issue by wenjie710** [mbiparva/spatiotemporal-multiplier-networks-pytorch#1]

**Comment by mbiparva** [comment]
Yes, using the MATLAB pre-trained networks ported to PyTorch, I can reproduce the original results. The thing is you need to fine-tuned hyper-parameters. I will update the repository with those hyper-parameters later.

**Comment by wtt55** [comment]
> _No description provided._



> 
Have you been test?I  apply to ask some questions with you,my connection995129257@qq.com
